mlonmcu/read_write_file_lock/read_write_filelock.py

```diff
@@ -0,0 +1,325 @@
+#
```

Comment:
I would rename the `read_write_lock` directory to something shorter.

Either just `mlonmcu/lock/` or move `mlonmcu/context.py` and `read_write_filelock.py` into a new subdirectory `mlonmcu/context/`